std/algorithm/sorting.d

```diff
@@ -1949,6 +1949,8 @@ sort!((a, b) => hashFun(a) < hashFun(b))(array);
 schwartzSort!(hashFun, "a < b")(array);
 ----
 
+See also $(WEB http://rosettacode.org/wiki/Sort_an_array_of_composite_structures#D, this example) on Rosetta Code.
+
```

Comment:
This should be reformatted to be
```
See Also:
$(WEB ...)
```
So it's is properly recognized and formatted.